module/input/Camera/src/AravisCamera.cpp

```diff
@@ -173,6 +173,12 @@ namespace input {
             context->reactor.emit<Scope::DIRECT>(msg);
             arv_stream_push_buffer(stream, buffer);
         }
+        else {
```

Comment:
I think I would prefer to this to be
```
if (buffer != NULL) {
    if (arv_buffer_get_status(buffer) == ARV_BUFFER_STATUS_SUCCESS) {
        // send message
        arv_stream_push_buffer(....); 
    }
    else {
        arv_stream_push_buffer(....);
    }
}
```

Whats your opinion @TrentHouliston ?

Reply:
I agree it should be changed to that, but I also think the compiler would probably optimise it to that anyway :P

Reply:
readability concerns are more important than potential compiler optimisations?

Reply:
Changes made, it matches the same format as the examples given in the documentation:
https://aravisproject.github.io/docs/aravis-0.4/ArvCamera.html

Reply:
> readability concerns are more important than potential compiler optimisations?

I said I agree!